### PR TITLE
Expand table options

### PR DESCRIPTION
**Pull request by Amy-Xu**
In this PR, the distribution table and difference table are generalized as I proposed in [this issue](https://github.com/OpenSourcePolicyCenter/webapp-public/issues/35).
 
- diagnostic table: function `create_distribution_table` is now able to present the difference between baseline and user-specified plan based on (AGI or _Expanded_Income) (deciles/bins).

- difference table: function `create_difference_table` can present OSPC tax, payroll (FICA), or the combined tax of the two.

@MattHJensen @talumbau  Could you review? If this is fine for Tax-Calculator, then I can expand dropq `utils.py` accordingly to further prepare for the webapp. @talumbau Let me know if you want me to do this and you  review afterward.

### CODE REVIEW

**Comment by MattHJensen** [comment]
@talumbau and @Amy-Xu, just flagging this again for your attention. 

cc @zrisher

**Comment by talumbau** [comment]
I think it would be appropriate to add a unit test for `add_columns`. The test could just pass in a simple Dataframe with columns `c04470`, `c00100`, `_standard` and `c09600`, and just a few rows with the difference relevant cases.

**Comment by Amy-Xu** [comment]
@talumbau I added the test as you suggested and merged this PR with master. Could you take a look again and merge this PR if it looks good?

File: taxcalc/utils.py [hunk 1]
```diff
@@ -379,13 +394,17 @@ def create_difference_table(calc1, calc2, groupby,
...
-    groupby, String object
+    calc1 :  the first Calculator object
+    calc2 : the other Calculator object
+    groupby : String object
         options for input: 'weighted_deciles', 'small_income_bins',
         'large_income_bins', 'webapp_income_bins'
         determines how the columns in the resulting DataFrame are sorted
-
+    income_measure : string
+        options for input: '_expanded_income', '_ospctax'
```
Comment (talumbau) [inline taxcalc/utils.py]: `_ospctax` is supposed to be `_iitax` now, correct?

File: taxcalc/utils.py [hunk 2]
```diff
@@ -379,13 +394,17 @@ def create_difference_table(calc1, calc2, groupby,
...
+        classifier of income bins/deciles
+    income_to_present : string
+        options for input: '_ospctax', '_fica', '_combined'
```
Comment (talumbau) [inline taxcalc/utils.py]: same comment as above.

**Pull request merged**